test/hotspot/jtreg/serviceability/jdwp/DebuggeeLauncher.java

```diff
@@ -107,12 +107,7 @@ public int getJdwpPort() {
 
     @Override
     public void onStringRead(StreamHandler handler, String line) {
-        if (handler.equals(errorHandler)) {
-            terminateDebuggee();
-            listener.onDebuggeeError(line);
-        } else {
-            processDebuggeeOutput(line);
-        }
+        processDebuggeeOutput(line);
```

Comment:
You might want to look at #4560, which added the onDebuggeeError() code, which is no longer being called from anywhere after this change. @alexmenkov

Reply:
This method existed from beginning. I'll remove it.
See: https://github.com/openjdk/jdk/commit/4f55b6c7e03687e9b799456d6e4d0b82027566ec